pitch.md

```diff
@@ -1,44 +1,160 @@
-# Project Title (replace me)
+# ComplexCalendar
 
 ## Overview
 _What's this project about, briefly?_
 
+This project is about making complex calendar's easy to make. For any calendar
+software it is easy to make specific events but for most people, their life
+heavily revolves around a core calendar that repeats and doesn't change often.
+I want to make it easier for users to define their complex calendars in fewer
+words. 
```

Comment:
This makes me super happy, like I said in class.  This is definitely a problem I've tried to deal with before.